dss/operations/lambda_params.py

```diff
@@ -0,0 +1,291 @@
+"""
+Get and set environment variables in deployed lambda functions using the SSM param store
+variable named "environment".
+"""
+import os
+import sys
+import select
+import json
+import argparse
+import logging
+import typing
+
+from botocore.exceptions import ClientError
+
+from dss.operations import dispatch
+from dss.operations.ssm_params import get_ssm_variable_prefix, fix_ssm_variable_prefix
+from dss.operations.ssm_params import get_ssm_parameter, set_ssm_parameter
```

Comment:
Looks like `get_ssm_parameter` needs to be removed.

Reply:
D'oh, I forgot that I needed set_ssm_parameter and unset_ssm_parameter in this file. I inlined those functions per your suggestion in PR #2432 . Thanks for pointing out the superfluous import.